mux.PGO.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we really need a new top level directory in the repo just for this one file?

Reply:
There is no top level directory, just a file in the top level.

---
In reply to: [301743348](https://github.com/microsoft/microsoft-ui-xaml/pull/988#discussion_r301743348) [](ancestors = 301743348)

Reply:
Weird. Why is CodeFlow showing this as a file in a folder named "mux.PGO.props"? 

---
In reply to: [302109935](https://github.com/microsoft/microsoft-ui-xaml/pull/988#discussion_r302109935) [](ancestors = 302109935,301743348)